bazel-mode.el

```diff
@@ -21,6 +21,35 @@
 ;;
 ;;; Code:
 
+(defgroup bazel-mode nil
+  "Major mode for Bazel BUILD files."
+  :link '(url-link "https://github.com/bazelbuild/emacs-bazel-mode")
+  :group 'languages)
+
+(defcustom bazel-mode-buildifier-cmd "buildifier"
+  "Filename of buildifier executable."
+  :type 'file
```

Comment:
nit: maybe add a link to https://github.com/bazelbuild/buildtools/tree/master/buildifier or so?

Reply:
Done. I expect this particular URL will remain valid for quite a while.